shared/src/main/scala/drt/shared/HasAirportConfig.scala

```diff
@@ -526,6 +526,6 @@ object AirportConfigs {
     "KNA" -> 83.8, "SOM" -> 90.6, "LCA" -> 89.3, "GRD" -> 105.9
   )
 
-  val allPorts: List[AirportConfig] = ema :: edi :: stn :: man :: ltn :: lhr :: lgw :: Nil
+  val allPorts: List[AirportConfig] = ema :: edi :: stn :: man :: ltn :: lhr :: lgw :: lgw.copy(portCode = "BHX"):: Nil
```

Comment:
This isn't correct.

Reply:
yeah, you'll need to create a separate config for BHX, you can probably copy one of the existing ones, and then you can get some of the details you need from V1 - let me know if you need login details for v1.